frontend/src/pages/hardwareProfiles/ManageNodeResourceSection.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { FormSection, Flex, FlexItem, Button } from '@patternfly/react-core';
+import { Identifier } from '~/types';
+import NodeResourceTable from './nodeResource/NodeResourceTable';
+import ManageNodeResourceModal from './nodeResource/ManageNodeResourceModal';
+
+type ManageNodeResourceSectionProps = {
+  nodeResources: Identifier[];
+  setNodeResources: (identifiers: Identifier[]) => void;
+};
+
+export const ManageNodeResourceSection: React.FC<ManageNodeResourceSectionProps> = ({
```

Comment:
This is a single file, maybe consider exporting it as `default` like other components?